core/src/main/java/com/digitalpebble/stormcrawler/protocol/AbstractHttpProtocol.java

```diff
@@ -96,4 +99,14 @@ protected static String getAgentString(String agentName,
         return buf.toString();
     }
 
+    /**
+     * Return new instance of {@link SimpleDateFormat} to format dates in HTTP
+     * headers, cf. <a href=
+     * "https://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.3">sec3.3 in
+     * RFC 2616</a>.
+     */
+    public static SimpleDateFormat getHttpDateFormat() {
```

Comment:
unless this is used by any of the code in the protocol package, this should be put somewhere else e.g. within the AdaptiveScheduler class itself?